pkg/api/http_server.go

```diff
@@ -66,8 +67,6 @@ func (hs *HTTPServer) Init() error {
 	hs.macaron = hs.newMacaron()
 	hs.registerRoutes()
 
-	session.Init(&setting.SessionOptions, setting.SessionConnMaxLifetime)
```

Comment:
Did some of the changes related to #16148 that seems appropriate for these changes